sqlglot/expressions.py

```diff
@@ -4793,6 +4793,11 @@ class List(Func):
     is_var_len_args = True
 
 
+# String pad, kind "L" -> LPAD, "R" -> RPAD
```

Comment:
Let's not do it this way, just use a boolean flag if there are only two variants